### PR TITLE
Added Questions Page

### PR DESCRIPTION
**Pull request by aniketxparihar**
Added Questions Page for Quiz App please Review

image

image

image

### CODE REVIEW

**Review by shraddha-amber — COMMENTED** [review]
You might want to change the text colours, they are quite light

**Comment by aniketxparihar** [comment]
@shraddha-1402 thanks for the suggestion. Changed the text Colors
image

**Review by shraddha-amber — APPROVED** [review]

File: questions/questions.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://atomui.netlify.app/css/component.css">
+    <link rel="stylesheet" href="questions.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Document</title>
```
Comment (Tarandeep-s1ngh) [inline questions/questions.html]: I guess, you forgot to change the title

File: questions/questions.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  margin-left: auto;
+}
+
+#questionNoAndScore,
+#question,
+#answer,
+#prevAndNext {
+  margin-bottom: 4rem;
+  width: 60%;
+}
```
Comment (Tarandeep-s1ngh) [inline questions/questions.css:41]: Suggestion: better to use class selectors over id selectors whenever possible.

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: questions/questions.html (hunk 1 above)
Comment (aniketxparihar) [reply questions/questions.html]: Thank You!

**Pull request merged**